src/libpgmoneta/json.c

```diff
@@ -844,6 +878,40 @@ fill_value(char* str, char* key, uint64_t* index, struct json* o)
    return 1;
 }
 
+static int
+handle_escape_char(char* str, uint64_t* index, uint64_t len, char* ch)
+{
+   uint64_t idx = *index;
+   idx++;
+   if (idx == len)   // security check
+   {
+      return 1;
+   }
+
+   // Check the next character after checking '\' character
+   switch (str[idx])
+   {
+      case '\'':
```

Comment:
This can go if the json standard allows it.

Reply:
We can get rid of this as well

Reply:
Yeah we probably should not include single quote in the logic at all. Instead we should probably error out when it's escaped. Perhaps you need to check all the other logic related to it. It's my bad, the initial comment in the issue is misleading, apologies :)

Reply:
No issue, let's get it done :)